wp-includes/widgets/class-wp-widget-media-image.php

```diff
@@ -317,10 +317,6 @@ public function render_control_template_scripts() {
 						echo sprintf( __( 'Current image: %s' ), '{{ data.currentFilename }}' );
 					?></p>
 				<# } #>
-			<# } else { #>
-				<div class="attachment-media-view">
-					<p class="placeholder"><?php echo esc_html( $this->l10n['no_media_selected'] ); ?></p>
```

Comment:
Actually, no. You can see that `isSelected` is looking at `control.model.get( 'error' )` and in this case here we need to pass the `error` into the template.